frontend/src/views/consumer/event-listing/TicketInputSet.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { FlexBoxHorizontal } from './EventBooking';
+import { observer } from 'mobx-react';
+import { clamp } from '../../../utils';
+import Form, { InputField } from '../../../components/form';
```

Comment:
I still think it's not a good idea to use relative paths in the imports.. if you eg. want to move this file to another folder, these won't work anymore..

Reply:
any idea how to set it up?

Reply:
Here's one way to do it: https://medium.com/@ktruong008/absolute-imports-with-create-react-app-4338fbca7e3d